.golangci.yml

```diff
@@ -150,3 +150,6 @@ linters-settings:
       - ptrToRefParam
       - typeUnparen
       - unnecessaryBlock
+  wrapcheck:
+    ignorePackageGlobs:
```

Comment:
I'm not very familiar with golangci: does this avoid linter errors in error returned that are not wrapped? If so, why do we need to disable it? Why can we not use `sce.WithMessage`?

Reply:
> does this avoid linter errors if the error returned is not wrapped? 

Yes

> Why can we not use `sce.WithMessage`?

The functions within `fileparser` already returns errors that use `sce.WithMessage`, so it seemed redundant to wrap it again. Essentially, the reason I added this was because `fileparser` is more an internal package and `wrapcheck` is meant to flag _external_ package errors that are not wrapped.